.github/workflows/security-scans.yml

```diff
@@ -200,6 +200,7 @@ jobs:
       matrix:
         image:
           - "functions-runtime"
```

Comment:
I guess we can remove functions-runtime now, right ?

Reply:
I would leave this in for the security scans so that we get scans for the old images as well just in case